bin/config.js

```diff
@@ -1,19 +1,31 @@
-const rc = require('rc')
+const { writeFile } = require('fs')
+const configDefaults = require('./files/configDefaults.json')
+const minimist = require('minimist')
 
-module.exports = rc('cordlr', {
-  plugins: [],
-  prefix: '$',
-  adminPerms: [
-    'KICK_MEMBERS',
-    'BAN_MEMBERS',
-    'ADMINISTRATOR',
-    'MANAGE_CHANNELS',
-    'MANAGE_GUILD',
-    'MANAGE_MESSAGES',
-    'MUTE_MEMBERS',
-    'DEAFEN_MEMBERS',
-    'MOVE_MEMBERS',
-    'MANAGE_NICKNAMES',
-    'MANAGE_ROLES_OR_PERMISSIONS'
-  ]
-})
+const configPath = `${process.cwd()}/package.json`
+let config
+try {
+  config = require(configPath)
+} catch (e) {
+  // file doesn't exist
+  config = { cordlr: configDefaults }
+}
+
+// insert command line options
+const args = minimist(process.argv.slice(2))
+for (const key of Object.keys(args)) {
```

Comment:
The reason i didn't use `Object.assign` is because i wanted to filter out the `_` property minimist might attach, since we have no way of interpreting those settings (yet)